CONTRIBUTING.md

```
@@ -0,0 +1,149 @@
+2. After authenticating with the Tinybird CLI, navigate to the `lib/tinybird` directory:
+   ```sh
+   cd lib/tinybird
+   ```
+3. Push the necessary data sources using the following command:
+   ```sh
+   tb push datasources/*
+   tb push endpoints/get_*
+   ```
+4. Don't forget to set the `TINYBIRD_TOKEN` with the appropriate rights in your `.env` file.
+
+#### Updating Tinybird
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)